tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -825,6 +826,22 @@ def test_forward_logical():
     test_logical_not()
 
 
+#######################################################################
+# Where, Select
+# --------------------
+def test_where():
+    ''' Where: return elements depending on conditions'''
+    with tf.Graph().as_default():
+        with tf.Session() as sess:
+            input1 = tf.placeholder(tf.int32, shape=[1, 4, 4, 3], name='input1')
+            input2 = tf.placeholder(tf.int32, shape=[1, 4, 4, 3], name='input2')
+            mask = input1 > input2
+            tf.where(mask, input1 + 1, input2 * 2)
+            in_data1 = np.random.uniform(0, 10, size=(1, 4, 4, 3)).astype("uint32")
```

Comment:
hope there is no reason to limit the dtype to "uint32" !

Reply:
yeah, unint32 was used here because the related dtype of placeholder is tf.int32. It works well with other dtype like float32/float64.